protocol/query/method/message/data/dataAddReaction.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://raw.githubusercontent.com/dedis/student_21_pop/master/protocol/query/method/message/data/dataAddReaction.json",
+  "description": "Match a add reaction query",
+  "type": "object",
+  "properties": {
+    "object": {
+      "const": "reaction"
+    },
+    "action": {
+      "const": "add"
+    },
+    "reaction_id": {
+      "description": "Code indicating a reaction",
+      "type": "string",
+      "$comment": "unicode of the reaction (ex: \"U+1F44D\" for like)"
+    },
```

Comment:
As per other comments, based on the description and Unicode terminology, I'd suggest renaming the field `reaction_codepoint`.

You might also want to clarify how you want the unicode codepoint to be expressed.
JSON strings support Unicode, you can natively express Unicode code points with `\uXXXX` escaping or the representation of the character itself (e.g., `😄`).
I'd advise against using a `U+XXXX` representation in a format (JSON) which already has a standard way of representing unicode.
It's best if the JSON-Schema $comment explicitly states which of the two standard JSON representations of Unicode is preferred.

Reply:
Thank you for the clarification. Is it better now?

Reply:
It's clearer. I'm not sure it's simpler for you to implement though 😅 
Do all the front-end / back-end teams agree that this is easier for them (collectively) to implement ? 
I'd have guessed that the character itself (e.g., 😄) would be a lot simpler, especially as escaping is not necessary in JSON and JSON serializers might not do that by default.

In any case, I'll approve the PR and let you discuss between yourselves as needed.

Reply:
I agree, as soon as I submitted the change I wanted to change it back for the character itself. I'll discuss this with the social media team. Thank you for the review.